### PR TITLE
Refactor/storybook ts jest fonts

### PR DESCRIPTION
**Pull request by german-schneck**
**Summary:**

- TypeScript has been properly configured 2.
- You have configured Jest + React Testing Library + Test in TypeScript base.
- Avenir Next font has been added to the project.
- Changed the appearance of the storybook.
- Removed the light-dark mode system as it requires a better implementation to the rest of the interface. (At the moment we only need dark mode)
- Moved the components created since 04/08/22.

### CODE REVIEW

**Comment by german-schneck** [comment]
**Summary:**

- TypeScript has been properly configured 2.
- You have configured Jest + React Testing Library + Test in TypeScript base.
- Avenir Next font has been added to the project.
- Changed the appearance of the storybook.
- Removed the light-dark mode system as it requires a better implementation to the rest of the interface. (At the moment we only need dark mode)
- Moved the components created since 04/08/22.

File: src/components/display/Table/components/PercentageIndicator/index.tsx [hunk 1]
```diff
@@ -3,14 +3,14 @@ import React from "react";
 import NumberFormat from "react-number-format";
 
 // Assets
-import ArrowChartUpIcon from "@assets/icons/arrow-chart-up-icon.svg";
-import ArrowChartDownIcon from "@assets/icons/arrow-chart-down-icon.svg";
+import ArrowChartUpIcon from "../../../../../assets/icons/arrow-chart-up-icon.svg";
+import ArrowChartDownIcon from "../../../../../assets/icons/arrow-chart-down-icon.svg";
```
Comment (Xzirez) [inline src/components/display/Table/components/PercentageIndicator/index.tsx:7]: Why do you need these relative imports? It looks really strange with "../../../". Can we make them absolute paths instead?

File: src/components/display/Table/components/Row/index.tsx [hunk 2]
```diff
@@ -1,12 +1,12 @@
 // Dependencies
 import React from "react";
+import { Row as _Row } from "react-table";
 
 // Assets
-import ExpandedArrowIcon from "@assets/icons/arrow-expansion-icon.svg";
+import ExpandedArrowIcon from "../../../../../assets/icons/arrow-expansion-icon.svg";
```
Comment (Xzirez) [inline src/components/display/Table/components/Row/index.tsx:6]: same here about imports

File: src/components/display/Table/components/Row/styles.ts [hunk 3]
```diff
@@ -1,6 +1,6 @@
 // Dependencies
 import styled from "styled-components";
-import { styledIf } from "utils/styled";
+import { styledIf } from "../../../../../utils/styled";
```
Comment (Xzirez) [inline src/components/display/Table/components/Row/styles.ts:3]: same

File: src/components/display/Table/components/TableButton/stories.tsx [hunk 4]
```diff
@@ -3,7 +3,7 @@ import React from "react";
 import { ComponentStory, ComponentMeta } from "@storybook/react";
 
 // Assets
-import EditPenIcon from "@assets/icons/edit-pen-icon.svg";
+import EditPenIcon from "../../../../../assets/icons/edit-pen-icon.svg";
```
Comment (Xzirez) [inline src/components/display/Table/components/TableButton/stories.tsx:6]: same about impots

File: src/components/display/Table/components/TableButton/styles.ts [hunk 5]
```diff
@@ -1,6 +1,6 @@
 // Dependencies
 import styled from "styled-components";
-import { Button } from "components/styled";
+import { Button } from "../../../../styled";
```
Comment (Xzirez) [inline src/components/display/Table/components/TableButton/styles.ts:3]: imports

File: src/components/display/Table/index.tsx [hunk 6]
```diff
@@ -4,7 +4,7 @@ import { useTable, useSortBy, Column } from "react-table";
 import { useClickAway } from "react-use";
 
 // Assets
-import ArrowBottomIcon from "@assets/icons/arrow-bottom-icon.svg";
+import ArrowBottomIcon from "../../../assets/icons/arrow-bottom-icon.svg";
```
Comment (Xzirez) [inline src/components/display/Table/index.tsx:7]: same

File: src/components/display/Table/styles.ts [hunk 7]
```diff
@@ -33,52 +33,54 @@ export const TableView = styled.table`
...
-        white-space: nowrap;
-        
-        &.action {
-          display: flex;
-          align-items: center;
-          justify-content: flex-end;
-        }
-      }
+
+  font-family: "Avenir", sans-serif;
```
Comment (Xzirez) [inline src/components/display/Table/styles.ts:37]: Is it supposed to be "Avenir" or "Avenir-next Redhat" ?

In design I think it was "Avenir-next"

@NataliaAvila-PM

**Comment by Xzirez** [comment]
Overall LGTM. Just minor comments.

Also this request is large. We should try to make them much smaller in future. I know this was a big refactor so I guess it's fine.

File: src/components/display/Table/styles.ts (hunk 7 above)
Comment (NataliaAvila-PM) [reply src/components/display/Table/styles.ts:37]: Hi! Yes, it should be Avenir Next as primary, if not Red Hat Text, and if not sans-serif
font-family: Avenir Next, Red Hat Text, sans-serif

**Comment by Xzirez** [comment]
One thing i forgot to mention:

Try to use describe to group the type of test we are using. Example here instead of writing "renders" for every it. I nest them inside an it and then I don't need to repeat myself for every test. This is also beneficial if we want different variables for different describe groups. See image below

Screenshot 2022-04-12 at 12 58 12

File: src/components/display/Table/components/PercentageIndicator/index.tsx (hunk 1 above)
Comment (german-schneck) [reply src/components/display/Table/components/PercentageIndicator/index.tsx:7]: At first we had somewhat unfinished relative routes.

On one side @assets/*, and on the other assets/*. Then in the new refactor we started using assets/*.

If you want I'll check the project and add absolute paths to the elements.

File: src/components/display/Table/styles.ts (hunk 7 above)
Comment (NataliaAvila-PM) [reply src/components/display/Table/styles.ts:37]: reviewing...

**Comment by german-schneck** [comment]
Hi Chris, it was because I made a blank copy of the project and did not look at the previous configuration (except for some parts).

If you want I can change it.

**Pull request merged**